Domain/Repositories/Repository.cs

```diff
@@ -68,6 +69,23 @@ public void RemoveMember(RepositoryMember repositoryMember)
             member.Delete();
         }
         
+        public void ThrowIfUserCantAccessRepositoryData(Guid userId)
+        {
+            if (!IsPrivate) return;
+            
+            var userIsRepositoryMember = Members.Any(mem => mem.Member.Id == userId && !mem.Deleted);
+            if (userIsRepositoryMember) return;
+            
+            var repositoryIsInOrganization = Organization != null;
+            if (!repositoryIsInOrganization) throw new RepositoryInaccessibleException();
```

Comment:
Could these 2 lines be simlified to if(Organization is null) throw new ...?

Reply:
yes but readability would be lower